index.js

```diff
@@ -38,3 +77,4 @@ function merge(leftArray, rightArray) {
 }
 
 console.log(mergeSort([-1, 3, 0, 12, 10, 100, -100]))
+>>>>>>> 37a5ed17204f63485919a3ac62d1d25b01169ec7
```

Comment:
you didn't resolve the conflicts here
<img src="https://cultofthepartyparrot.com/parrots/hd/hmmparrot.gif"  width="20"/>